lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -48,6 +48,24 @@ module Style
       #     private def baz; end
       #
       #   end
+      # @example AllowModifiersOnSymbols: true
```

Comment:
The formatting here seems off.

Reply:
Hmm, I see now it's messed up in the entire file. I guess you just stuck to the existing examples.

Reply:
Do you mean the empty line before `@example`?

Reply:
That and the empty lines between `bad/good`. Just compare this to other source files and you'll see what I mean.

Reply:
Fixed.